lib/elixir/test/elixir/kernel/expansion_test.exs

```diff
@@ -1818,14 +1818,14 @@ defmodule Kernel.ExpansionTest do
 
   describe "bitstrings" do
     test "nested match" do
-      assert expand(quote(do: <<?-, (<<_, _::binary>> = rest())::binary>>))
+      assert expand(quote(do: <<?-, <<_, _::binary>> = rest()::binary>>))
```

Comment:
@whatyouhide @josevalim should not we keep the parentheses? 🤔

Reply:
Mh, I guess we should parenthesize every time we have an operator on the LHS of `::`?